lldb/source/Target/StackFrame.cpp

```diff
@@ -71,7 +72,7 @@ StackFrame::StackFrame(const ThreadSP &thread_sp, user_id_t frame_idx,
   // recursive functions properly aren't confused with one another on a history
   // stack.
   if (IsHistorical() && !m_cfa_is_valid) {
-    m_id.SetCFA(m_frame_index);
+    m_id.SetCFA(m_frame_index, thread_sp->GetProcess().get());
```

Comment:
Is process ever going to be nullptr here? Wondering if a process reference can be used throughout this change.

Reply:
Good question. I actually started with a reference, but realized that I was not able to prove we can always dereference the process here. AFAICT all calls to this constructor come from StackFrameList code, and there the _thread_ is always non-null.

But the thread itself only contains a weak pointer to the process